index.js

```diff
@@ -97,7 +97,9 @@ module.exports = function (config) {
     // since we parse the whole feature file at start
     removeNotExecutedScenarios();
 
-    fs.writeFile(path.join(global.output_dir, config.outputFile), JSON.stringify(allFeatures, null, 2), (err) => {
+    const filename = config.uniqueFileNames ? `cucumber_output_${Math.floor(new Date().getTime() / 1000)}.json` : config.outputFile;
```

Comment:
Thanks for this addition. The only thing i would change is to add uniqueFileName to any user specified filename if true

```
let filename;
if (config.uniqueFileNames) filename = `${config.outputFile}_${Math.floor(new Date().getTime() / 1000)}.json`;
```

maybe something like this?